dist/session.js

```diff
@@ -157,7 +160,7 @@ session.delimiter = function (str) {
   if (str === undefined) {
     return this._delimiter;
   }
-  this._delimiter = String(str).trim() + ' ';
```

Comment:
The prompt delimiter is a very touchy subject, and if you look in the issue history, there's a lot of subtle bug fixes involved with it. Be careful before changing things, as seeming fixes can break a lot of other things in this area.